.github/workflows/notify_discord.yml

```
@@ -0,0 +1,102 @@
+on:
+  pull_request:
+    types: [opened, closed]
+    branches:
+      - master
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)